src/api/entities/SecurityToken/Offerings.ts

```diff
@@ -46,9 +46,9 @@ export class Offerings extends Namespace<SecurityToken> {
   /**
    * Retrieve all of the Token's Offerings. Can be filtered using parameters
    *
-   * @param opts.status - status of the offerings to fetch
+   * @param opts.status - status of the offerings to fetch. As long as the STO has one of the passed statuses, it will be returned
```

Comment:
This comment is not accurate. STO must have all the passed statuses to be returned.

Reply:
I had this in place before we spoke, I forgot to change it 😅